src/dex-ui/store/poolsSlice/utils.ts

```diff
@@ -39,26 +40,31 @@ interface CalculateUserPoolMetricsParams {
  */
 const calculateUserPoolMetrics = (params: CalculateUserPoolMetricsParams): UserPool => {
   const { poolTokenBalances, userTokenBalances, userTokenPair, fee } = params;
-  const { pairToken, tokenA, tokenB } = userTokenPair;
+  const { tokenA, tokenB } = userTokenPair;
   const totalVolumeLocked = calculateTotalValueLockedForPool({
     poolTokenBalances,
-    tokenAAccountId: tokenA.accountId,
-    tokenBAccountId: tokenB.accountId,
+    tokenAAccountId: tokenA.tokenMeta.tokenId ?? "",
+    tokenBAccountId: tokenB.tokenMeta.tokenId ?? "",
   });
   /* TODO: Get real LP Amounts for pools */
-  const mockedLPPoolTokenBalances = appendLiquidityTokenBalance(poolTokenBalances, totalVolumeLocked);
-  const percentOfPool = calculatePercentOfPool({
+  // const mockedLPPoolTokenBalances = appendLiquidityTokenBalance(poolTokenBalances, totalVolumeLocked);
```

Comment:
Same here - remove instead of commenting out.

Reply:
Done